From 8f14b076bef29daa1764f0e5e72094cd495e45fa Mon Sep 17 00:00:00 2001
From: Bertrand Lorentz <bertrand.lorentz@gmail.com>
Date: Thu, 27 Aug 2009 19:29:24 +0200
Subject: [PATCH 28/33] add Samsung Galaxy as a USB music player

It's an Android phone, so keys are copied from the HTC Android G1.
Extracted from http://bugzilla.gnome.org/show_bug.cgi?id=589029
---
 .../10freedesktop/10-usb-music-players.fdi         |   13 +++++++++++++
 1 files changed, 13 insertions(+), 0 deletions(-)

diff --git a/fdi/information/10freedesktop/10-usb-music-players.fdi b/fdi/information/10freedesktop/10-usb-music-players.fdi
index f0b62ae..3a3d352 100644
--- a/fdi/information/10freedesktop/10-usb-music-players.fdi
+++ b/fdi/information/10freedesktop/10-usb-music-players.fdi
@@ -364,6 +364,19 @@
 	    <append key="portable_audio_player.audio_folders" type="strlist">Recorded Files/Voice/</append>
 	    <append key="portable_audio_player.playlist_path" type="string">Playlists/%File</append>
 	  </match>
+	  <!-- Samsung Galaxy, Android Phone -->
+          <match key="@storage.originating_device:usb.product_id" int_outof="0x6601;0x6603">
+             <addset key="portable_audio_player.access_method.protocols" type="strlist">storage</addset>
+             <append key="portable_audio_player.output_formats" type="strlist">audio/x-ms-wma</append>
+             <append key="portable_audio_player.output_formats" type="strlist">audio/3gpp</append>
+             <append key="portable_audio_player.output_formats" type="strlist">audio/mp4</append>
+             <append key="portable_audio_player.output_formats" type="strlist">audio/aac</append>
+             <append key="portable_audio_player.output_formats" type="strlist">audio/vnd.rn-realaudio</append>
+             <append key="portable_audio_player.output_formats" type="strlist">audio/x-midi</append>
+             <append key="portable_audio_player.output_formats" type="strlist">audio/x-wav</append>
+             <append key="portable_audio_player.output_formats" type="strlist">application/ogg</append>
+             <append key="portable_audio_player.audio_folders" type="strlist">Music/</append>
+          </match>
 	</match>
  	
 	<!-- Sony -->
-- 
1.7.1

